argocd/resource_argocd_project.go

```diff
@@ -60,10 +68,14 @@ func resourceArgoCDProjectCreate(d *schema.ResourceData, meta interface{}) error
 	tokenMutexProjectMap[projectName].RUnlock()
 
 	if err != nil {
-		switch strings.Contains(err.Error(), "NotFound") {
-		case true:
-		default:
-			return err
+		if strings.Contains(err.Error(), "NotFound") {
```

Comment:
This inverts the previous logic, resulting in a failure to create projects. It should be:
```suggestion
		if !strings.Contains(err.Error(), "NotFound") {
```

Reply:
Done ✅